bmc/redfish.go

```diff
@@ -342,14 +344,29 @@ func (r *RedfishBMC) checkBiosAttributes(attrs map[string]string) (reset bool, e
 	return
 }
 
-func (r *RedfishBMC) GetStorages(systemUUID string) ([]Storage, error) {
+func (r *RedfishBMC) GetStorages(ctx context.Context, systemUUID string) ([]Storage, error) {
 	system, err := r.getSystemByUUID(systemUUID)
 	if err != nil {
 		return nil, err
 	}
-	systemStorage, err := system.Storage()
+	var systemStorage []*redfish.Storage
+	err = wait.PollUntilContextTimeout(
+		ctx,
+		10*time.Second,
+		5*time.Minute,
```

Comment:
should we make interval and timeout configurable like in `bmcClient.WaitForServerPowerState`?

Reply:
I was thinking about it. We just have so many config options already :)